templates/layout.html

```diff
@@ -102,8 +102,12 @@
         <p class="text-muted">
           Donate CDNJS $5 on <a href="https://www.bountysource.com/teams/cdnjs" target="_blank">Bountysource</a> or become a contributor on <a href="https://github.com/cdnjs/cdnjs" target="_blank"><i class="fa fa-github"></i> GitHub</a> to make the project better and better!
           <span class="pull-right">
+            <a href="https://status.cdnjs.com" target="_blank" class="status">
+              <span class="status-color-dot"></span>
+              <span class="status-color-description"></span>
+            </a>
             <a href="https://twitter.com/cdnjs" target="_blank"><i class="fa fa-twitter-square"></i> Twitter</a>
-            &nbsp; <a href="https://cdnjs.discourse.group/" target="_blank"><i class="fa fa-comments"></i> Discourse</a>
+            <a href="https://cdnjs.discourse.group/" target="_blank"><i class="fa fa-comments"></i> Discourse</a>
```

Comment:
this is not a related change, is it?

Reply:
@PeterDaveHello As part of putting the indicator in the footer, I have dropped this extra spacing between the two old links to provide more consistent spacing between all items in the footer.